src/ansi.rs

```diff
@@ -668,6 +668,16 @@ pub trait Handler {
 
     /// Set hyperlink.
     fn set_hyperlink(&mut self, _: Option<Hyperlink>) {}
+
+    /// Set mouse cursor.
+    ///
+    /// The requested cursor shapes usually have a name from CSS specificiation
+    /// <https://www.w3.org/TR/css-ui-3/#cursor>.
```

Comment:
I think `usually` phrasing is a bit odd here. Of course a client can pick whatever, but it's a bit too uncertain.

I think it would be better to phrase it more like this:

```suggestion
    /// Clients can specify any string, but legal values are defined in the CSS specificiation:
    /// <https://www.w3.org/TR/css-ui-3/#cursor>.
```

But there's really not that many values available. Why wouldn't we just define an enum? That would make everything **so** much easier for consumers.